internal/scorecard/alpha/tests/basic.go

```diff
@@ -31,5 +31,21 @@ func CheckSpecTest(bundle registry.Bundle) scapiv1alpha2.ScorecardTestResult {
 	r.State = scapiv1alpha2.PassState
 	r.Errors = make([]string, 0)
 	r.Suggestions = make([]string, 0)
+
+	crSet, err := GetCRs(bundle)
+	if err != nil {
+		r.Errors = append(r.Errors, "error getting custom resources")
+		r.State = scapiv1alpha2.FailState
+		return r
+	}
+
+	for _, cr := range crSet {
+		if cr.Object["spec"] == nil {
+			r.Errors = append(r.Errors, "error spec does not exist")
+			r.State = scapiv1alpha2.FailState
+			return r
+		}
+	}
+
```

Comment:
It will change by https://github.com/operator-framework/operator-sdk/pull/3061 however shows fine move forward with. 

c/c @varshaprasad96

Reply:
Ill create a follow up PR after this gets merged for reformatting.

Reply:
Hi  @theishshah  and @varshaprasad96.
 
Since the #3061 is merged, and it is not passing in the CI then, I think that we can just rebase it with the b master and add the new check as done in #3061

Reply:
Passing now .. so it can be a follow-up 👍

Reply:
Added this test in #3097